common/persistence/sql/storage/sqldb/uuid.go

```diff
@@ -0,0 +1,93 @@
+// Copyright (c) 2018 Uber Technologies, Inc.
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+
+package sqldb
+
+import (
+	"database/sql/driver"
+	"encoding/hex"
+
+	"github.com/google/uuid"
+)
+
+// UUID represents a 16-byte universally unique identifier
+// this type is a wrapper around google/uuid with the following differences
+//  - type is a byte slice instead of [16]byte
+//  - db serialization converts uuid to bytes as opposed to string
+type UUID []byte
+
+// NewUUIDFromString returns a new UUID parsed from the given string
+// returns nil if the input is empty string
+// panics if the given input is malformed
+func NewUUIDFromString(s string) UUID {
```

Comment:
I don't like this name as it looks like it generates UUID from any string. 
I would just use the same name as uuid package: MustParse.

Reply:
3 options:
 (1) Either I create a sqldb/uuid subpackage and name the method MustParse - so for caller, it will look like uuid.MustParse() - same as uuid package
(2) Or name this method sqldb.ParseUUID(s string) - sqldb.MustParse() is weird
(3) Or name this method sqldb.StringToUUID(s string)

I am inclined towards option (2) - because it avoids import namespace collisions in other parts of code

Reply:
renamed to MustParseUUID()